framework/materials/multi_group_xs/openmc_import.cc

```diff
@@ -137,17 +137,17 @@ MultiGroupXS::Initialize(const std::string& file_name,
     }
 
     // Chi
-    auto chi = H5ReadDataset1D<double>(file, path + "chi");
-    OpenSnLogicalErrorIf(chi.empty(), "\"chi\" data block not found in " + file_name + ".");
-    if (not chi.empty())
+    chi_ = H5ReadDataset1D<double>(file, path + "chi");
+    OpenSnLogicalErrorIf(chi_.empty(), "\"chi\" data block not found in " + file_name + ".");
+    if (not chi_.empty())
```

Comment:
No need for `if (not chi_.empty())` since the code won't get past the `OpenSnLogicalErrorIf` if chi is empty.

Reply:
fix. I suppose this was there before :-)